apps/web/lib/features/team-members-kanban-view.tsx

```diff
@@ -16,15 +16,78 @@ import {
 
 export const KanbanView = ({ kanbanBoardTasks }: { kanbanBoardTasks: IKanban }) => {
 	const {
+		data: items,
 		columns: kanbanColumns,
 		updateKanbanBoard,
 		updateTaskStatus,
 		isColumnCollapse,
-		reorderStatus
+		reorderStatus,
+		addNewTask
 	} = useKanban();
 
-	const [items, setItems] = useState<IKanban>(kanbanBoardTasks);
-
+	// const [items, setItems] = useState<IKanban>({
```

Comment:
@Anishali2 not sure why we need this code commented, we should either move it to separate file or remove or uncomment and use etc

Reply:
Sorry for making these stupid mistakes it won't happen again I'll review the PR before pushing.